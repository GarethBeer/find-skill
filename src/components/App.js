import React from "react";
import { Switch, Route } from "react-router-dom";

import NavBar from "./NavBar";
import Skills from "./Skills";
import AddSkills from "./AddSkills";

import "../styles/App.css";

function App() {
  return (
    <div className="App">
      <NavBar />
      <Switch>
        <Route exact path="/" component={Skills}>
          <Skills />
        </Route>
        <Route exact path="/add-skills" component={AddSkills}>
          <AddSkills />
        </Route>
      </Switch>
    </div>
  );
}

export default App;
