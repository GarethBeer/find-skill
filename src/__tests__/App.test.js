import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import App from "../components/App";

describe("App", () => {
  it("renders correctly to match the App snapshot", () => {
    const { asFragment } = render(
      <BrowserRouter>
        <App />
      </BrowserRouter>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
