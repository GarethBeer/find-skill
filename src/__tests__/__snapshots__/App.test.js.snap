// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly to match the App snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <ul
        class="navbar-links"
      >
        <li
          class="navbar-links-item"
        >
          <a
            href="/"
          >
            View Skills
          </a>
        </li>
        <li
          class="navbar-links-item"
        >
          <a
            href="/add-skills"
          >
            Add a Skill
          </a>
        </li>
      </ul>
    </div>
    <div
      class="skills"
    >
      <div>
        <div>
          Loading...
        </div>
      </div>
      <div
        class="skillcards"
      />
    </div>
  </div>
</DocumentFragment>
`;
